SIT.Manager/SIT.Manager.csproj

```diff
@@ -47,6 +47,9 @@
             <AutoGen>True</AutoGen>
             <DependentUpon>Resources.resx</DependentUpon>
         </Compile>
+        <Compile Update="Views\CrashWindow.axaml.cs">
+          <DependentUpon>CrashWindow.axaml</DependentUpon>
+        </Compile>
```

Comment:
This seems to make Visual Studio look off for me and don't think it needs it

Reply:
This was generated by visual studio and I have no idea what it does